### PR TITLE
W 11855274/syaml render clone format

### PR DESCRIPTION
**Pull request by llibarona**

### CODE REVIEW

File: als-actions/shared/src/test/resources/actions/documentFormatting/json/expected/innerNode.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "parent": {
-    "key": "value"
-  },
+  "key": "value"
+},
```
Comment (BlockLucas) [inline als-actions/shared/src/test/resources/actions/documentFormatting/json/expected/innerNode.json:4]: shouldn't this one (line 4) have a tab?

File: als-actions/shared/src/test/resources/actions/documentFormatting/json/expected/multiple-syntax-errors.json [hunk 2]
```diff
@@ -1,17 +1,18 @@
...
     {
       "somewhat": "valid",
       "valid": true
     },
     "entry"
   ],
   "paths": {
-    "dd"
-    dd
+    "dd": null
```
Comment (BlockLucas) [inline als-actions/shared/src/test/resources/actions/documentFormatting/json/expected/multiple-syntax-errors.json:15]: This line (15) and the previous one with "syntax ": "error" on line 5 are producing different outputs, not sure how to check that from source, are those changes ok?

File: als-actions/shared/src/test/resources/actions/documentFormatting/json/expected/syntax-error-sequence.json [hunk 3]
```diff
@@ -4,9 +4,9 @@
       "syntax " "error"
     },
     {
-      "somewhat": "valid",
-      "valid": true
-    },
+  "somewhat": "valid",
+  "valid": true
+},
```
Comment (BlockLucas) [inline als-actions/shared/src/test/resources/actions/documentFormatting/json/expected/syntax-error-sequence.json]: are these changes on indent expected?

File: als-server/shared/src/test/resources/actions/documentFormatting/expected/raml.raml [hunk 4]
```diff
@@ -1,58 +1,56 @@
...
         type: flight
-    responses: 
-      200: 
-        body: 
-          application/json: myNumber
+    responses:
+      200:
+        body:
+          application/json:
+            myNumber
```
Comment (BlockLucas) [inline als-server/shared/src/test/resources/actions/documentFormatting/expected/raml.raml:56]: weird jump line? is minutia for me, it should be ok regardless

File: als-server/shared/src/test/resources/actions/documentRangeFormatting/expected/oas.json [hunk 5]
```diff
@@ -6,14 +6,14 @@
...
-    ], "responses": {"200": {"description": "OK", "content": {"application/json": {"schema": {"properties": {"myProp": {"type": "string"}}}}}}}}}
+                                                        {
+                                                          "in": "header",
+                                                          "schema": {
+                                                            "description": "Schema",
+                                                            "type": "string"
+                                                          },
+                                                          "name": "param"
+                                                        }
+                                                      ], "responses": {"200": {"description": "OK", "content": {"application/json": {"schema": {"properties": {"myProp": {"type": "string"}}}}}}}}}
```
Comment (BlockLucas) [inline als-server/shared/src/test/resources/actions/documentRangeFormatting/expected/oas.json:17]: check

**Review by emilianoascona — APPROVED** [review]

**Review by emilianoascona — APPROVED** [review]

**Pull request merged**